coast/plot_util.py

```diff
@@ -7,14 +7,14 @@
 
 import matplotlib.pyplot as plt
 from warnings import warn
-from .logging_util import get_slug, debug, info, warn, error
+from .logging_util import warn
 import numpy as np
 
 
 def r2_lin(x, y, fit):
     """For calculating r-squared of a linear fit. Fit should be a python polyfit
     object"""
-
+    # TODO Can someone sciencey give me the proper names for these?
```

Comment:
Not sure about some of these, there aren't always obvious names

fity = fit(y), perhaps y_estimate
diff = difference between real y and y_estimate. difference?
ybar = mean of y, as in the mathematical bar symbol
ymybar = mean_square_deviation
SStot = total_deviation
SSred = residual
R2 = correlation_coefficient R^2

Reply:
Have made these changes.